python/ray/experimental/iter.py

```diff
@@ -238,6 +239,46 @@ def combine(self, fn: Callable[[T], List[U]]) -> "ParallelIterator[U]":
         it.name = self.name + ".combine()"
         return it
 
+    def local_shuffle(self, shuffle_buffer_size: int,
+                      seed: int = None) -> "ParallelIterator[T]":
+        """Remotely shuffle items of each shard independently
+
+        Arguments:
+            shuffle_buffer_size (int): The algorithm fills a buffer with
+            shuffle_buffer_size elements and randomly samples elements from
+            this buffer, replacing the selected elements with new elements.
```

Comment:
```suggestion
                this buffer, replacing the selected elements with new elements.
```

Subsequent lines should be indented. Same comment applies in a number of places.